queries.sql

```diff
@@ -1,3 +1,58 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-12-12' and '2019-12-12';
+
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.50;
+
+SELECT * FROM animals WHERE neutered = true;
+
+SELECT * FROM animals WHERE name != 'Gabumon';
+
+SELECT * FROM animals WHERE weight_kg >= 10.40 and weight_kg <= 17.30;
+
+/*transactions*/
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE name NOT LIKE '%mon';
```

Comment:
- Kindly change this queries because the question it asks to update the animals table by setting the species column to pokemon for all animals that don't have species already set. it means `WHERE species IS NULL;` :+1: